cmd/clusterctl/client/cluster/cert_manager.go

```diff
@@ -56,6 +45,21 @@ const (
 
 	certManagerImageComponent = "cert-manager"
 	timeoutConfigKey          = "cert-manager-timeout"
+
+	certmanagerVersionAnnotation = "certmanager.clusterctl.cluster.x-k8s.io/version"
+	certmanagerHashAnnotation    = "certmanager.clusterctl.cluster.x-k8s.io/hash"
+
+	// NOTE: // If the cert-manager.yaml asset is modified, this line **MUST** be updated
+	// accordingly else future upgrades of the cert-manager component will not
+	// be possible, as there'll be no record of the version installed.
+	embeddedCertManagerManifestVersion = "v0.16.0"
```

Comment:
We should use v0.16.1 given that it was just released

Reply:
If it is ok for you I will send a separate PR as soon as this one merge

Reply:
I was going to suggest a separate PR because that way we can test out what updating the cert-manager manifest would look like.